### ISSUE TITLE
Suggestion: Find in single active file

### ISSUE THREAD

**Issue by rkpatel33** [tomrijndorp/vscode-finditfaster#26]
I wondering if this was considered and not implemented for any reason? It would be a nice addition and make this have the same features as the FZF plugin available for Neovim, which has a buffer search mode.

(Again, this is a game changer for me and making me come back to VsCode from Nvim.)

**Comment by tomrijndorp** [comment]
I'm not sure what this means, it sounds like you're describing "search text", that is, Ctrl+F so clearly I must be misunderstanding something :) Do you have a reference that shows me what the feature is about? I tried searching but all the search terms are quite generic so if you can pass me a link I'd be sure we're talking about the same thing. Thanks!

**Comment by rkpatel33** [comment]
You are right, ctrl-f will do this, but the benefit is of this is you get the list of all hits in a single file, as well as the code preview for each hit.

For example, here is Neovim FZF's "buffer search", which is what I am talking about. It's a nice UX.

<img width="1674" alt="image" src="https://user-images.githubusercontent.com/3799489/221779112-6e38d7b2-c176-4d8a-a7b1-352527abf199.png">

**Comment by tomrijndorp** [comment]
Ah I see, thanks for the explanation! I don't have a ton of time to work on the extension right now, but I'm going to leave this ticket open as a nice to have and might be able to give this a try later. Thanks for the suggestion!

**Comment by rkpatel33** [comment]
If I have some time I will make an attempt at making the adjustment in a PR.

**Comment by sahellebusch** [comment]
This feature would be awesome 😄
